apps/wallet/src/ui/app/pages/accounts/manage/AccountGroup.tsx

```diff
@@ -66,14 +69,24 @@ export function AccountGroup({
 						</Heading>
 						<div className="h-px bg-gray-45 flex flex-1 flex-shrink-0" />
 						<ButtonOrLink
-							onClick={(e) => {
-								// prevent button click from collapsing section
+							onClick={async (e) => {
 								e.stopPropagation();
-								// todo: implement me
+								try {
+									// todo: support "Add New" for more account types
+									if (type === 'mnemonic') {
+										createAccountMutation.mutate({
```

Comment:
`mutate` isn't `async` so same advice for using the `onSuccess` and `onError` callbacks